Makefile

```diff
@@ -0,0 +1,16 @@
+GO_SOURCES = $(shell find pkg -type f -name '*.go')
+
+vendor: glide.lock
+	glide install -v --force
+
+glide.lock: glide.yaml
+	glide up -v --force
+
+test: $(GO_SOURCES) vendor
+	kafka-topics --delete --zookeeper localhost:2181 --topic test-topic
```

Comment:
This fails the build if the topic doesn't exist.
Should prefix with a `-` or `|| true`

Reply:
@ericbottard Good catch. Also, it turns out the topic creation command can fail if the topic already exists (in spite of the fact it has been marked for deletion), so I prefixed that with a `-` too.

Reply:
This requires `delete.topic.enable` in the kafka config file, something that is not likely to be set for the common case. Wonder if we should rely on a randomized topic name instead? (separate PR)

Reply:
@ericbottard https://github.com/projectriff/message-transport/issues/5 raised to cover this.